packages/runtime-core/src/helpers/useModel.ts

```diff
@@ -118,9 +120,17 @@ export function useModel(
 }
 
 export const getModelModifiers = (
+  instance: ComponentInternalInstance,
   props: Record<string, any>,
   modelName: string,
 ): Record<string, boolean> | undefined => {
+  if (
+    __COMPAT__ &&
+    isCompatEnabled(DeprecationTypes.COMPONENT_V_MODEL, instance)
```

Comment:
This `__COMPAT__ && isCompatEnabled(DeprecationTypes.COMPONENT_V_MODEL, instance)` check is repeated in a few places. It'd be nice if that were done only once, but I guess that might affect dead code elimination?

Reply:
Solved in the latest push. Resolving.